chart/compass/charts/instance-creator/values.yaml

```diff
@@ -0,0 +1,23 @@
+deployment:
+  minReplicas: 1
```

Comment:
As per product standards we should have at least 2 replicas in order to ensure high availability